.gitignore

```diff
@@ -7,9 +7,19 @@
 *.png
 *.log
 *.pdf
+*.txt
+
+*.fits
+
+notes.txt
+
+mynotebooks/
```

Comment:
Some of these rules look specific to your directories so probably don't belong here.